### DISCUSSION TITLE
AoA indicator

### DISCUSSION THREAD

**Discussion by falkeblikket** [nguyenquyhy/Flight-Tracker-StreamDeck#248] [Q&A]
The Flight-Tracker-StreamDeck does not seem to support angle of attack yet. The SimConnect SDK lists this variable as ANGLE_OF_ATTACK_INDICATOR (Units:Radians, Type:TYPE_SINT16). Is there an easy way to implement this into this plugin or does it require a lot of programming?

**Comment by nguyenquyhy** [comment]
Please create and issue for this. I'll add in the next update.

**Reply by falkeblikket** [reply]
Thanks! :-D
